src/patient-registration/section/death-info/death-info-section.component.tsx

```diff
@@ -14,9 +14,11 @@ export const DeathInfoSection: React.FC<DeathInfoSectionProps> = ({ values }) =>
       <h5 className={`omrs-type-title-5 ${styles.formSectionTitle}`}>Death Info</h5>
       <section className={styles.fieldGroup}>
         <Input type="checkbox" label="Is Dead" name="isDead" />
-        {values.isDead && <Input type="date" label="Date of Death" name="deathDate" />}
         {values.isDead && (
-          <SelectInput options={['Unknown cause of death', 'stroke']} label="Cause of Death" name="deathCause" />
+          <>
+            <Input type="date" label="Date of Death" name="deathDate" />
+            <SelectInput options={['Unknown cause of death', 'stroke']} label="Cause of Death" name="deathCause" />
```

Comment:
Just a stylistic comment, for the `SelectInput` options, we should follow sentence case. So here we would have "Unknown cause of death" and "Stroke". We also don't need to specify that it's a cause of death as that is already implied. So we could just have it as "Unknown" and "Stroke".

Reply:
fixed :)